entc/gen/template/privacy/privacy.tmpl

```diff
@@ -35,52 +39,22 @@ var (
 	}
 {{- end }}
 
-type decisionCtxKey struct {}
-
-// DecisionContext creates a decision context.
-func DecisionContext(parent context.Context, decision error) context.Context {
-	if decision == nil || errors.Is(decision, Skip) {
-		return parent
-	}
-	return context.WithValue(parent, decisionCtxKey{}, decision)
-}
-
-func decisionFromContext(ctx context.Context) (error, bool) {
-	decision, ok := ctx.Value(decisionCtxKey{}).(error)
-	if ok && errors.Is(decision, Allow) {
-		decision = nil
-	}
-	return decision, ok
-}
+var (
+	// DecisionContext creates a new context from the given parent context with
+	// a policy decision attach to it.
+	DecisionContext = privacy.DecisionContext
```

Comment:
Since go doesn't support `const` we shouldn't use `var` but a forward function instead.